server/db/relationships.js

```diff
@@ -3,11 +3,13 @@ const Order = require('./models/Order')
 const Product = require('./models/Product')
 const ProductOrders = require('./models/ProductOrders')
 const Category = require('./models/Category')
+const User = require('./models/User')
+const Review = require('./models/Review')
 
 //once Product is imported
 //Category.belongsToMany(Product,  { through: 'productcategories' });
 //Product.belongsToMany(Category,  { through: 'productcategories' })
 
```

Comment:
Could you please add the relationship for your models here
```suggestion
User.hasMany(Review)
Product.hasMany(Review)

Review.belongsTo(Product)
Review.belpongsTo(User)
```